docs/sources/configure-client/opentelemetry/ebpf-profiler.md

```diff
@@ -0,0 +1,50 @@
+---
+title: OpenTelemetry Profiles Support (Experimental)
+menuTitle: OpenTelemetry Profiles
+description: Pyroscope's experimental support for OpenTelemetry profiles
+weight: 10
+---
+
+# OpenTelemetry profiles support (experimental)
+
+Pyroscope now includes experimental support for receiving and visualizing profiles from OpenTelemetry sources. This integration allows you to:
```

Comment:
We have some standard wording for experimental features: https://grafana.com/docs/writers-toolkit/write/shortcodes/#docsexperimental

Is the product name correct in the line below?

```suggestion
{{< docs/experimental product="OpenTelemetry profiles" >}}

Pyroscope includes experimental support for receiving and visualizing profiles from OpenTelemetry sources. This integration allows you to:
```